Test/AbstractBlockServiceTestCase.php

```diff
@@ -97,4 +97,20 @@ protected function assertSettings(array $expected, BlockContextInterface $blockC
 
         $this->assertSame($completeExpectedOptions, $blockSettings);
     }
+
+    /**
+     * NEXT_MAJOR: Remove this method when dropping support for < PHPUnit 5.4.
+     *
+     * @param string $class
+     *
+     * @return \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected function createMock($class)
```

Comment:
What about some base class to avoid code duplication? You can create something like [this](https://github.com/sonata-project/SonataAdminBundle/pull/4288/commits/6ee98abefe3f791e23669fc8c2de3033c50d3769) and avoid this method in each test class.